src/status_im/utils/utils.cljs

```diff
@@ -10,11 +10,10 @@
    (.alert (.-Alert rn-dependencies/react-native)
            title
            content
-           (clj->js
-            (vector (merge {:text                "OK"
-                            :style               "cancel"
-                            :accessibility-label :cancel-button}
-                           (when on-dismiss {:onPress on-dismiss}))))
+           (clj->js (vector (merge {:text                "OK"
```

Comment:
Are those purely indentation changes?

Reply:
[Lifting my top-level comment (see below) to here]

 yes, the `cljfmt` check rejected some unrelated existing code and I had to do a bit of reformatting to get my commit through.

Reply:
I've just ran `lein cljfmt check src/status_im/utils/utils.cljs` and it shows no formatting errors. Please revert all unrelated formatting changes.